alphagov/paas-uaa-customized-boshrelease
# Visual updates and a11y fixes to login ( and co.) pages

## What

This is a host of changes to visually update login and associated pages to latest styles and fix some accessibility issues. 



## How to review

- review commit by commit
- deploy to your environment
- check if all is still working

## Who can review

- not @kr8n3r 

## Changes (visual)

### new header layout (using product name)
<img width="829" alt="Screenshot 2020-06-16 at 10 41 16" src="https://user-images.githubusercontent.com/3758555/84758960-03c34f80-afbe-11ea-927c-0580c2f85f50.png">

### new focus states
<img width="834" alt="Screenshot 2020-06-16 at 10 38 50" src="https://user-images.githubusercontent.com/3758555/84758650-a4653f80-afbd-11ea-8c98-97cea5f37b92.png">

### error-summary focus on load and position above H1, updated title
<img width="595" alt="Screenshot 2020-06-16 at 10 42 46" src="https://user-images.githubusercontent.com/3758555/84759083-2bb2b300-afbe-11ea-834c-a1087fad1907.png">
